requirements.txt

```diff
@@ -4,86 +4,80 @@
 #
 #    pip-compile --output-file requirements.txt requirements.in
 #
-appdirs==1.4.0            # via setuptools
+asn1crypto==0.22.0        # via cryptography
 babel==2.3.4
 babeldjango==0.2.2
-boto3==1.4.4
-botocore==1.5.4           # via boto3, s3transfer
-braintree==3.34.0         # via django-payments
-cffi==1.9.1               # via cryptography
-cryptography==1.7.1       # via django-payments
-cssselect==1.0.1          # via premailer
-cssutils==1.0.1           # via premailer
+boto3==1.4.5
+botocore==1.5.92          # via boto3, s3transfer
+braintree==3.37.2         # via django-payments
+certifi==2017.7.27.1      # via requests
+cffi==1.10.0              # via cryptography
+chardet==3.0.4            # via requests
+cryptography==2.0.2       # via django-payments
 dj-database-url==0.4.2
 dj-email-url==0.0.10
-django-absolute==0.3      # via django-emailit
 django-bootstrap3==8.2.1
 django-cache-url==1.3.1
 django-countries==4.1
-django-emailit==0.2.2
-django-materializecss-form==1.0.1
 django-mptt==0.8.7
 django-payments==0.11.0.3
 django-prices-openexchangerates==0.1.12
 django-prices==0.4.13
-django-redis==4.7.0
-django-storages==1.5.2
+django-redis==4.8.0
+django-render-block==0.5  # via django-templated-email
+django-storages==1.6.5
+django-templated-email==2.2.0
 django-versatileimagefield==1.6.2
-django-webpack-loader==0.4.1
-django==1.10.7
-djangorestframework==3.5.3
+django-webpack-loader==0.5.0
+django==1.11.4
+djangorestframework==3.6.3
 docutils==0.13.1          # via botocore
 elasticsearch==2.4.1
+enum34==1.1.6             # via cryptography
```

Comment:
This file was generated using Python 2. Please rebuild using Python 3 as that's the default version we're targeting.